stormdemo/source/cpp/src/KaaDemo.cpp

```diff
@@ -72,7 +72,7 @@ int main()
     auto kaaClient =  Kaa::newClient();
 
     // Set a custom strategy for uploading logs.
-    kaaClient->setLogUploadStrategy(std::make_shared<LogUploadStrategy>());
+    kaaClient->setLogUploadStrategy(std::make_shared<LogUploadStrategy>(kaaClient->getKaaClientContext()));
```

Comment:
Please, get rid of the LogUploadStrategy class and use RecordCountLogUploadStrategy(1, kaaClient->getKaaClientContext()) instead.
And add corresponding header.